app/views/posts/index.html.erb

```diff
@@ -1,11 +1,50 @@
-<h1>Posts</h1>
+<%= stylesheet_link_tag 'posts_index' %>
 
+<div class="user-show-container">
+  <div class="user-image-container">
+    <img src="<%= @user.photo %>" alt="<%= @user.name %>">
+  </div>
+
+  <div class="user-details-container">
+    <h1><%= @user.name %></h1>
+    <p class="post-count">Number of posts: <%= @user.posts.count %></p>
+  </div>
+</div>
```

Comment:
- [optional] Hi @Xghayor, it is a good practice to keep our codebase DRY by avoiding repeating a piece of codes across views files. You can utilized Rails partial for this specific :+1: